src/warnet/deploy.py

```diff
@@ -221,3 +251,43 @@ def deploy_namespaces(directory: Path):
         finally:
             if temp_override_file_path.exists():
                 temp_override_file_path.unlink()
+
+
+def is_windows():
+    return sys.platform.startswith("win")
```

Comment:
doesn't need to be addressed in this PR , but this is code smell to me and indicates we really should stop doing port forwarding in favour of ingress (see https://github.com/bitcoin-dev-project/warnet/discussions/529)